code/script.js

```diff
@@ -1,53 +1,166 @@
-  // be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
+	// The if statement checks if the sender is the user and if that's the case it inserts
+	// an HTML section inside the chat with the posted message from the user
+	if (sender === "user") {
+		chat.innerHTML += `
+			<section class="user-msg">
+				<div class="bubble user-bubble">
+					<p>${message}</p>
+				</div>
+				<img src="assets/user.svg" alt="User" />
+			</section>
+		`;
+		// The else if statement checks if the sender is the bot and if that's the case it inserts
+		// an HTML section inside the chat with the posted message from the bot
+	} else if (sender === "bot") {
+		chat.innerHTML += `
+			<section class="bot-msg">
+				<img src="assets/boba.svg" alt="Bot" />
+				<div class="bubble bot-bubble">
+					<p>${message}</p>
+				</div>
+			</section>
+		`;
+	}
+	// This little thing makes the chat scroll to the last message when there are too many to
+	// be shown in the chat box
+	chat.scrollTop = chat.scrollHeight;
+	myPlay(bump);
+};
 
 // A function to start the conversation
 const greetUser = () => {
-  // Here we call the function showMessage, that we declared earlier with the argument:
-  // "Hello there, what's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, what's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆 and see what happens
+	myPlay(welcome);
+	showMessage("Hello there, what's your name?", "bot");
+};
+
+const handleInput = (event) => {
+	event.preventDefault();
+	// Store the value in a variable so we can access it after we
+	// clear it from the input
+	const userInput = document.getElementById("user-input").value;
+	showMessage(userInput, "user");
+	document.getElementById("user-input").value = "";
+	userName = userInput;
+	// After 1 second, show the next question by invoking the next function.
+	// passing the name into it to have access to the user's name if we want
+	// to use it in the next question from the bot.
+	setTimeout(() => showFoodOptions(), 1000);
+};
+
+const showFoodOptions = () => {
+	showMessage(
+		`Hi ${userName} Please choose your tea base`,
+		"bot"
+	);
+
+	// add class hidden to input and submit button
+	document.getElementById("user-input").classList.add("hidden");
+	document.querySelector(".send-btn").classList.add("hidden");
+	//add buttons with options
+	buttonTea.classList.add("button-tea");
+	greenTea.textContent = "Green Tea";
+	blackTea.textContent = "Black Tea";
+	milkTea.textContent = "Milk Tea";
+	buttonTea.append(greenTea, blackTea, milkTea);
+	form.appendChild(buttonTea);
+};
+
+const handleFoodChoice = (choice) => {
+	showMessage(` ${choice}`, "user");
+	teaChoice = choice;
+	buttonTea.removeChild(blackTea);
+	buttonTea.removeChild(milkTea);
+	buttonTea.removeChild(greenTea);
+	setTimeout(() => toppingChoice(choice), 500);
+};
+
+const toppingChoice = (choice) => {
```

Comment:
Maybe this function could be called showToppingOptions to follow the previous pattern?